src/js/handlers/handleCalculator.js

```diff
@@ -0,0 +1,66 @@
+import {
+  INITIAL_VALUE,
+  ERROR_MESSAGES,
+  OPERATORS,
+} from "../utils/constants.js";
+import { TOTAL } from "../utils/constants.js";
+import { $ } from "../utils/dom.js";
+import { operation } from "./operation.js";
+
+
+let numberCount = 0;
+
+export const handleCalculator = ({ target }) => {
+  switch (target.classList[0]) {
+    case "digit":
+      handleDigit(target.textContent);
+      return;
+    case "operation":
+      handleOperation(target.textContent);
+      return;
+    case "modifier":
+      $(TOTAL).textContent = INITIAL_VALUE;
+      numberCount = 0;
+      return;
+  }
+};
+
+const handleDigit = ($digit) => {
+  if (numberCount >= 3) {
+    alert(ERROR_MESSAGES.INVALID_DIGIT_LENGTH);
+    throw new Error(ERROR_MESSAGES.INVALID_DIGIT_LENGTH);
+  }
+
+  if ($(TOTAL).textContent === INITIAL_VALUE) {
+    $(TOTAL).textContent = $digit;
+    numberCount++;
+  } else {
+    $(TOTAL).textContent += $digit;
+    numberCount++;
+  }
+};
+
+const handleOperation = ($operator) => {
+  if ($(TOTAL).textContent === INITIAL_VALUE) {
+    alert(ERROR_MESSAGES.INVALID_INPUT);
+    throw new Error(ERROR_MESSAGES.INVALID_DIGIT_LENGTH);
+  }
+
+  if ($operator === "=") {
+    getResult();
+    numberCount = 0;
+    return;
+  }
+  $(TOTAL).innerText += $operator;
+
+  return;
+
+};
+
+const getResult = () => {
+  const operator = $(TOTAL)
+    .textContent.split("")
+    .find((i) => OPERATORS.includes(i));
```

Comment:
```suggestion
  const operator = [ ...$(TOTAL).textContent ].find(OPERATORS.includes.bind(OPERATORS));
```
이렇게 표현할 수도 있답니다!
사실 별 차이 없네요 ㅋㅋ 편한 방법으로 사용하시면 될 것 같습니다.

Reply:
앗 코드가 한결 간결해졌군요! ㅋㅋ 이렇게 쓸 수 도 있겠네요 감사합니다!

Reply:
이 부분은 `.bind(OPERATORS)`를 빼보면 알 수 있을 것 같아요!